src/components/style/interactive/previewCard.css

```diff
@@ -0,0 +1,83 @@
+.fdsCard {
+  color: inherit;
+  text-decoration: none;
+  width: 215px;
+  border-bottom-left-radius: var(--border-radius-default);
+  border-bottom-right-radius: var(--border-radius-default);
+  box-shadow: 0 0 2px 0 rgba(0, 0, 0, 0.12), 0 2px 2px 0 rgba(0, 0, 0, 0.24);
+  background-color: var(--color-white);
+}
+
+.fdsCard-cover {
+  height: 130px;
+  position: relative;
+  padding: var(--space-half);
+}
+
+.fdsCard-cover--purple {
+  background-color: var(--color-purple);
+}
+
+.fdsCard-cover--aqua {
+  background-color: var(--color-aqua);
+}
+
+.fdsCard-cover--darkPurple {
+  /* are these one off colors, are these standards, can we replace with standards, etc */
+  background-color: #783983;
+}
+
+.fdsCard-cover--pink {
+  /* are these one off colors, are these standards, can we replace with standards, etc */
+  background-color: #D3346F;
+}
+
+.fdsCard-cover-hover {
+  display: none;
+  text-align: center;
+  padding: 0 var(--space-half);
+  position: absolute;
+  top: 0;
+  left: 0;
+  right: 0;
+  bottom: 0;
```

Comment:
Not now, but if we find ourselves doing this a lot (position absolute and setting all sides to zero) we could make a utility class for that. Not now, but just something to note.